cmd/kube-proxy/app/server.go

```diff
@@ -477,6 +477,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 			return nil, fmt.Errorf("unable to read IPTables MasqueradeBit from config")
 		}
 
+		nodeIP, err := getNodeIP(client, hostname)
+		if err != nil {
+			return nil, fmt.Errorf("unable to fetch node IP: %v", err)
```

Comment:
There is no such flag. That is why I think it may be better to grab the bind-address here.